### PR TITLE
Statistics info block

### PR DESCRIPTION
**Pull request by ArtemAnoshin**

### CODE REVIEW

File: cleantalk/antispam/acp/main_module.php [hunk 1]
```diff
@@ -125,6 +127,9 @@ function main($id, $mode)
 			'CLEANTALK_STATS__SFW_NETS'     => $config['cleantalk_stats__sfw_nets'],
 			'CLEANTALK_DEBUG'               => $config['cleantalk_debug'] ? $config['cleantalk_debug'] :'',
             'CLEANTALK_STATS__LAST_SPAM_REQUEST_TIME' => $config['cleantalk_stats__last_spam_request_time'] ? date('M d Y H:i:s', $config['cleantalk_stats__last_spam_request_time']) : 'unknown',
+            'CLEANTALK_STATS__AVERAGE_REQUEST_TIME' => ($stat_requests && $stat_requests[min(array_keys($stat_requests))]['average_time'])
```
Comment (safronik) [inline cleantalk/antispam/acp/main_module.php:141]: isset() is required here.

File: cleantalk/antispam/acp/main_module.php [hunk 2]
```diff
@@ -10,6 +10,8 @@
 
 namespace cleantalk\antispam\acp;
 
+use cleantalk\antispam\library\Cleantalk\Common\API;
```
Comment (safronik) [inline cleantalk/antispam/acp/main_module.php:13]: Why this commit is here?
It doesn't relate to statistics in any way. It  should be in some other branch.

**Review by safronik — CHANGES_REQUESTED** [review]

File: cleantalk/antispam/acp/main_module.php (hunk 1 above)
Comment (ArtemAnoshin) [reply cleantalk/antispam/acp/main_module.php:141]: Done

File: cleantalk/antispam/acp/main_module.php (hunk 2 above)
Comment (ArtemAnoshin) [reply cleantalk/antispam/acp/main_module.php:13]: Yes, but so it happened

**Pull request merged**